client-frameworks-support/client-support-ui5/src/ui5-support-lib.js

```diff
@@ -0,0 +1,107 @@
+sap.ui.define(['sap/ui/model/json/JSONModel', '@luigi-project/client/luigi-client'], function(JSONModel, LuigiClient) {
+  const model = new JSONModel();
+  LuigiClient.addInitListener(ctx => {
+    model.setData(ctx);
+  });
+
+  LuigiClient.addContextUpdateListener(ctx => {
+    model.setData(ctx);
+  });
+
+  return {
+    connectTo: function(oComponent) {
+      const routingConfig = oComponent.getManifestEntry('sap.ui5').routing;
+      oComponent.setModel(model, '$luigiCtx');
+
+      // Create preload view
+      sap.ui.define('luigi/DummyView', ['sap/ui/core/mvc/View', 'sap/ui/core/Control'], function(View) {
```

Comment:
We should rediscuss the namespace we used for the `DummyView` -  it should be aligned with the `ui5-support-lib` - so the we do not pollute any namespace.